config/cache_test.go

```diff
@@ -23,9 +24,10 @@ func TestTerragruntConfigCacheOperation(t *testing.T) {
 
 	testCacheKey := "super-safe-cache-key"
 
-	cache := cache.NewCache[TerragruntConfig]()
+	ctx := context.Background()
+	cache := cache.NewCache[TerragruntConfig]("TerragruntConfig")
```

Comment:
I would use constants everywhere, for example we could reuse them in unit tests.
```suggestion
	cache := cache.NewCache[TerragruntConfig](TerragruntConfigCacheName)
```

Reply:
I meant to use the constant 

```go
const  TerragruntConfigCacheName = "TerragruntConfig"
```